frontend/src/reviews/ReviewComments.tsx

```diff
@@ -31,23 +40,76 @@ export default function ReviewComments({ release, accessRequest }: ReviewComment
     ...semverOrAccessRequestIdObject,
   })
 
-  const reviewDetails = useMemo(
-    () =>
-      reviews.map((review) =>
-        review.responses.map((response) => <>{response.decision && <ReviewDecision response={response} />}</>),
-      ),
-    [reviews],
-  )
+  const reviewDetails = useMemo(() => {
+    let decisionsAndComments: Array<ReviewResponseKind> = []
+    reviews.forEach((review) => {
+      review.responses.forEach((response) => decisionsAndComments.push(response))
+    })
+    if (release && release.comments) {
+      decisionsAndComments = [...decisionsAndComments, ...release.comments]
+    }
+    decisionsAndComments.sort(sortByCreatedAtAscending)
+    return decisionsAndComments.map((response) => {
+      if (isReviewResponse(response)) {
+        return <ReviewDecisionDisplay key={response.createdAt} response={response as ReviewResponse} />
+      } else {
+        return <ReviewCommentDisplay key={response.createdAt} response={response as ReviewComment} />
+      }
+    })
+  }, [reviews, release])
+
+  async function submitReviewComment() {
+    if (release && currentUser) {
+      const updatedRelease: ReleaseInterface = release
+      if (!updatedRelease.comments) {
+        updatedRelease.comments = []
+      }
+      const newComment: ReviewComment = {
+        comment: newReviewComment,
+        user: currentUser.dn,
+        createdAt: new Date().toISOString(),
+      }
+      updatedRelease.comments.push(newComment)
+      const res = await putRelease(updatedRelease)
+      if (res.ok) {
+        mutateRelease()
+        setNewReviewComment('')
+      } else {
+        setCommentSubmissionError(await res.json())
+      }
+    } else {
+      setCommentSubmissionError('There was a problem submitting this comment, please try again later.')
+    }
+  }
 
   if (isReviewsError) {
     return <MessageAlert message={isReviewsError.info.message} severity='error' />
   }
 
+  if (isCurrentUserError) {
+    return <MessageAlert message={isCurrentUserError.info.message} severity='error' />
+  }
+
   return (
     <>
       {reviews.length > 0 && <Divider />}
-      {isReviewsLoading && <Loading />}
+      {isReviewsLoading && isCurrentUserLoading && <Loading />}
```

Comment:
You still display the below stack even when the user is loading.  If a person submits a comment there the code will break.

The button should prevent duplicate clicks with a loading icon or similar.

These text fields should likely use markdown rendering / a markdown editor.